daemon/api_system_volumes.go

```diff
@@ -318,6 +326,39 @@ func postSystemVolumesActionReplaceRecoveryKey(c *Command, req *systemVolumesAct
 	return AsyncResponse(nil, chg.ID())
 }
 
+func postSystemVolumesActionReplacePlatformKey(c *Command, req *systemVolumesActionRequest) Response {
+	// XXX: Only support passphrase auth mode for now?
+	if req.AuthMode == "" {
```

Comment:
shouldn't we check that this is not PIN and that req.PIN and req.Passphrase are not provided at the same time?

Reply:
Good point, I updated the core volume options validation and used that